### DISCUSSION TITLE
自定义completions唤出列表的排序方式

### DISCUSSION THREAD

**Discussion by sentimental114** [imzbf/md-editor-rt#146] [Q&A]
请问如何自定义completions唤出列表的排序方式

**Comment by sentimental114** [comment]
@imzbf

**Reply by imzbf** [reply]
这个定义不了，都是在这里按照顺序设定的，它作为一个codemirror的扩展被添加到其中，可以考虑通过config移除默认的这个扩展，然后按照你的顺序自定义

**Reply by sentimental114** [reply]
感谢。在codemirror的文档里已经找到解决方案，在自己的数组调用map方法生成options的时候添加boost，这样就能按照自定义的顺序展示了。
